### ISSUE TITLE
[DropDowns] NVDA disables arrow keys navigation shortcuts when grouping is enabled

### ISSUE THREAD

**Issue by zstoyanova9** [telerik/kendo-angular#4449]
### Description

When grouping is configured in a DropDowns component and the NVDA screen reader is turned on, the default keyboard navigation of the arrow keys stops working and the items cannot be navigated. 

### Steps To Reproduce

1. Configure a DropDowns component with [grouping enabled](https://www.telerik.com/kendo-angular-ui/components/dropdowns/dropdownlist/grouping#grouping) (for example DropDownList).
2. Turn on the NVDA screen reader.
3. Focus the component and open its popup.
4. Try to navigate through the items using the Up and Down arrow keys.

StackBlitz example - https://stackblitz.com/edit/angular-p6vybj

### Screenshots or video

https://github.com/user-attachments/assets/0b02c77b-d5ea-467a-83a5-7e7188ea05d8

### Actual Behavior

The items in the popup cannot be navigated with the arrow keys when grouping is enabled and the NVDA screen reader is turned on.

### Expected Behavior

The NVDA screen reader should not affect the default behavior of the arrow keys keyboard navigation shortcut when grouping is configured. 

### Last working version of the Kendo UI for Angular package (if regression).

v15.4.0

Example with v15.4.0 - https://stackblitz.com/edit/angular-p6vybj-cfm8d5

**Comment by slavenai** [comment]
Fixed in `v18.0.0`.
